### ISSUE TITLE
Feature Request: Ethernet/EtherCAT communication protocol

### ISSUE THREAD

**Issue by Aquaharmonics** [vedderb/bldc#364]
To integrate with modern Real-Time (RT) systems, having a hardware option to communicate with Ethernet/EtherCAT based systems would be very powerful.

**Comment by vedderb** [comment]
Agree, I will look into this at some point when I have hardware with Ethernet.
